src/version/C/version_ciface.F90

```diff
@@ -0,0 +1,41 @@
+! THIS VERSION: GALAHAD 5.1 - 2024-08-16 AT 10:30 GMT.
+
+#include "galahad_modules.h"
+#include "galahad_cfunctions.h"
+
+!-*-*-*-*-*-  G A L A H A D _  V E R S I O N    C   I N T E R F A C E  -*-*-*-*-
+
+!  Copyright reserved, Gould/Orban/Toint, for GALAHAD productions
+!  Principal authors: Jaroslav Fowkes & Nick Gould
+
+!  History -
+!    originally released GALAHAD Version 5.1. August 16th 2024
+
+!  For full documentation, see
+!   http://galahad.rl.ac.uk/galahad-www/specs.html
+
+  MODULE GALAHAD_VERSION_ciface
+
+    USE GALAHAD_VERSION, ONLY: f_version_galahad => version_galahad
+
+  CONTAINS
+
+!  --------------------------------------
+!  C interface to fortran version_galahad
+!  --------------------------------------
+
+    SUBROUTINE version_galahad( major_version, minor_version, patch_version )  &
```

Comment:
Should we add a suffix `_s` like the other routines in single precision?
-> `version_galahad_s`